### PR TITLE
Clean up logits in phi3 C++ example

### PR DESCRIPTION
**Pull request by daijh**
Remove the logits demo code as it contains errors and lacks necessary C language implementation

### CODE REVIEW

**Comment by daijh** [comment]
Sorry not provide more detail about the PR.

Build genai with 4b12d57dfbc0097bf3f1b9c7660e2e79f02ff003.

> Creating tokenizer...
Prompt: (Use quit() to exit) Or (To terminate current output generation, press Ctrl+C)
some prompt
Generating response...
**Session Terminated: Unsupported Device type: 4**

It happens in 
```c++
std::unique_ptr<OgaTensor> output_logits = generator->GetOutput("logits");
```

As `output_logits` is not actually used, so simply removing it to avoid exception.

**Review by baijumeswani — APPROVED** [review]

**Comment by baijumeswani** [comment]
Since that part of the code is not really doing anything, I think it makes sense to remove it from the example. @ajindal1 Maybe we can document in another way.

[daijh](https://github.com/daijh) Thanks for the contribution.
